app/templates/macros/nav_macros.html

```diff
@@ -29,9 +29,20 @@
 {% macro account_items(current_user) %}
     <a href="{{ url_for('main.about') }}" class="item">About</a>
     <a href="{{ url_for('contact.index') }}" class="item">Contact Us</a>
+    <div class="ui simple dropdown item">
+        Information 
+        <i class="dropdown icon"></i>
+        <div class="menu">
+            <h3 class="ui header">Info</h3>
```

Comment:
I think we can actually remove this header. We needed it for the 'Manage Resources' since we had different categories we can manage but here everything fall under 'Information'